lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-storage_manager.rb

```diff
@@ -0,0 +1,7 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_StorageManager < MiqAeServiceExtManagementSystem
+    expose :parent_manager,         :association => true
+    expose :cloud_volumes,          :association => true
+    expose :cloud_volume_snapshots, :association => true
```

Comment:
I'm far from an expert in the automation code - but shouldn't these three lines (or at least the last two) be in the cinder_manager file and not the storage_manager file?  I would expect to add another file for the swift_manager, and would not think that :cloud_volumes and :cloud_volume_snapshots would be exposed to Swift.